config/main.py

```diff
@@ -1770,21 +1771,26 @@ def startup(ctx, interface_name):
         if interface_name is None:
```

Comment:
if get_interface_naming_mode() == "alias":
        interface_name = interface_alias_to_name(interface_name)

@stevenlu99 If the interface naming mode is "alias" mode, this change will break the command ? as the API  interface_alias_to_name() don't handle interface range still ?